source/renderer/app/components/settings/categories/StakePoolsSettings.js

```diff
@@ -192,6 +205,16 @@ export default class StakePoolsSettings extends Component<Props, State> {
 
   handleIsValid = (url: string) => url === '' || SMASH_URL_VALIDATOR.test(url);
 
+  handleErrorMessage = (value: string) => {
+    const { intl } = this.context;
+    let errorMessage = messages.invalidUrl;
+    const { pathname, search } = getUrlParts(value);
+    if (!/^https:\/\//i.test(value)) errorMessage = messages.invalidUrlPrefix;
```

Comment:
Just a small thing but I wonder if that `if/else` statement (without curly braces) passes linting?

Reply:
@DominikGuzei it does :)

Reply:
interesting, maybe we should adjust our linting rules 😆

Reply:
😅 I find it much easier to visualize simple checks like these without braces, but can add them :)